app-frontend/src/app/pages/admin/organization/organization.html

```diff
@@ -1,10 +1,13 @@
 <div class="container-admin-header">
   <div class="admin-header">
     <div class="admin-logo image-placeholder editable-image"
-         ng-if="($ctrl.fetching || !$ctrl.orgLogoUri.length) && $ctrl.isSuperOrAdmin"
+         ng-if="$ctrl.fetching"
+    ></div>
+    <div class="admin-logo image-placeholder editable-image-placeholder"
+         ng-if="!$ctrl.orgLogoUri.length && $ctrl.isSuperOrAdmin"
          ng-click="$ctrl.addLogoModal()"
     ></div>
-    <div class="admin-logo editable-image"
+    <div class="admin-logo editable-image-always"
          ng-if="$ctrl.orgLogoUri.length && $ctrl.isSuperOrAdmin"
```

Comment:
I would add `!$ctrl.fetching` here.

For the next `<div>`, I would do something like `ng-if="!$ctrl.fetching && !$ctrl.orgLogoUri.length && !$ctrl.isSuperOrAdmin"`

And for the one after the next `<div>`, I would add `!$ctrl.fetching` to the `ng-if` as well.

I think now it should cover all the potential cases.